readme.md

```diff
@@ -0,0 +1,16 @@
+#Project64  
+![Project64](http://i.imgur.com/8b6Qono.png)
+###A Nintendo 64 Emulator  
+
+###Licence  
+Licenced under GPLv2 (See [Licence.md](https://github.com/project64/project64/blob/master/licence.md))
+
+###Downloading
+You can download the latest Public Stable build [Here](http://www.pj64-emu.com/downloads/project64/binaries/)  
+If you would like a Not-So-Stable build, you can either Build it yourself (See Building) or use the unofficial build [Here](http://hyvart.com/download/project64/project64.application)
+
+
+###Building  
+Building Project64 requires (at this point in time) a copy of Visual Studio 2008 or newer, 2015 Recommended, running on XP or newer (Visual Studio 2015 will not run on Windows XP).  
```

Comment:
> (Visual Studio 2015 will not run on Windows XP)

I'd suggest leaving that comment out. The compatibility matrix for Visual Studio is not Project64's domain. I think indicating the compatible versions for the project is enough.
Also, it may be unclear whether that means that VS version won't run on XP, or a VS2015-compiled Project64 won't run on XP (which is false).
Finally, it would be more accurate to say that it's any version of VS starting with 2012 what won't run on XP.

> (at this point in time)

I'd leave this out. When we get a running UNIX version, there will surely be a big announcement, and this document will be updated accordingly.